examples/helloworld/lib/templates/frames/index.pug

```diff
@@ -0,0 +1,17 @@
+//-
+ Copyright 2014 Skytap Inc.
```

Comment:
It looks like the copyright date changed her. was that intentional?

Reply:
It was not intentional. I changed the date back!